Tesla.css

```diff
@@ -0,0 +1,477 @@
+  white-space: nowrap;
+  margin: 0 10px;
+}
+
+.nav__model {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 37%;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+
+.nav__input {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 14%;
+}
+
+.nav__link:hover {
+  color: #E51937;
+}
+
+.nav__link-s::after {
+  content:'';
+  display: block;
+  width: 100%;
+  height: 2px;
+  background: #E51937;
+  margin-top: 6px;
+}
+
+ /* .nav.nav__link:hover::after {
+  content:'';
+  display: block;
+  width: 60px;
+  height: 2px;
+  background: #E51937;
+  margin-top: 6px;
+} */
+
+.background {
+  background: url(img/background-header.png);
+  background-repeat: no-repeat;
+  max-width: 1514px; 
+  background-size: contain;
+  background-position: center center;
+  margin: 0 auto;
+}
+
+.title {  
+  width: 371px;
+  margin: 0 auto;
+  margin-bottom: 26px;
+}
+
+.title__h1 {
+  margin: 0 auto;
+  margin-top: 74px;
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: 9px;
+}
+
+.title__h2 {
+  margin: 0 auto;
+  font-size: 15px;
+  font-weight: 500;
+  text-align: center;
+}
+
+.image__car {
+  display: block;
+  size: contain;
+  width: 67%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 26px;
+}
+
+.box {
+  margin: 0 auto;
+  margin-bottom: 37px;
+  max-width: 800px;
+  width: 56%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.box__item {
+  display: flex;
+  align-items: center;
+}
+
+.box__img-1 {
+  width: 46px;
+  height: 34.13px;
+  margin-right: 14px;
+  padding: 0;
+}
+
+.box__img-2 {
+  width: 30px;
+  height: 50px;
+  margin-right: 11px;
+}
+
+.test__img-3 {
+  width: 22px;
+  height: 22px;
+}
+
+.box__item-1 {
+  margin-right: 83px;
+}
+
+.box__item-2 {
+  margin-right: 105px;
+}
+
+.box__text{
+  display: inline-block;
+  font-weight: 600;
+}
+
+.box__text-speed {
+  font-size: 12px;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  margin: 0;
+}
+
+.box__text-number {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.box__text-sec {
+  font-size: 20px;
+  margin-left: 6px;
+}
+
+.mobile-box {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.mobile-test {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.test {
+  margin-right: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.test__img-3 {
+  margin-top: 20px;
+  margin-bottom: 16px;
+  margin-right: 7px;
+  width: 22px;
+  height: 22px;
+}
+
+.test__text {
+  margin: 0;
+  margin-top: 23px;
+  margin-bottom: 17px;
+  text-transform: uppercase;
+  font-size: 13px;
+  font-weight: bold;
+  white-space: nowrap;
+}
+
+.vector {
+  width: 28px;
+  margin: 0 auto 93px;
+}
+
+@media screen and (max-width: 1200px) {
+  .nav__model {
+      display: none;
+  }
+  .box {
+      flex-wrap: wrap;
+  }
+  .mobile-box {
+      margin-bottom: 40px;
+  }
+}
+
+.main {
+    max-width: 1920px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)